Tests/UnitTests/APIToRequestMappingTests.swift

```diff
@@ -199,4 +211,11 @@ class APIToRequestMappingTests: XCTestCase {
         }
         XCTAssert(try session.size(window: "myWindow") == (width: 500, height: 500))
     }
+
+    func testMaximizeSelectedWindow() throws {
+        let mockWebDriver: MockWebDriver = MockWebDriver()
+        let session: Session = Session(webDriver: mockWebDriver, existingId: "mySession")
+        mockWebDriver.expect(path: "session/mySession/window/myWindow/maximize", method: .post)
+        try session.maximize(windowHandle: "myWindow")
```

Comment:
This doesn't really verify that the window has been maximized.

Reply:
I think this is a test we'll have to refactor after adding in some kind of sample UI. I commented on an issue #110, and that'll give us a space to know that it filled out.

Reply:
This file is for tests of how the API maps to HTTP requests so this is fine. It would be nice to have tests with a real app but that would go in a different file.